Formula/r/rom-tools.rb

```diff
@@ -34,9 +34,6 @@ class RomTools < Formula
   uses_from_macos "zlib"
 
   on_linux do
-    depends_on "portaudio" => :build
-    depends_on "portmidi" => :build
-    depends_on "pulseaudio" => :build
```

Comment:
any upstream refs for removing these build deps?

Reply:
this would affect the build that not to be able to use system dependencies as shown below. in that sense, we should keep these lines.